### PR TITLE
Add version attribute as action output

### PR DESCRIPTION
**Pull request by Poeschl**
During using the information addon, I was missing the version attribute. This PR adds it as additional output.

### CODE REVIEW

**Comment by frenck** [comment]
I can see this being helpful for some add-ons structures, but in a CI/CD automated environment, it doesn't make much sense to me.

Versions should be defined by thinks like releases or git tags.

**Comment by Poeschl** [comment]
I will use it as tags for my addons which is indeed not a good practice for releasing them. Since its still a freetime-project I have currently no motivation to come up with a complex release structure or splitted add-on repositories.

The version is also an basic attribute of an addon as well which could be used also in CI. Thinking about checks of matching versions of Changelog and config.json and automatic tag creation on master pushes.

**Review by frenck — CHANGES_REQUESTED** [review]
👍 

But needs adjustments to the README/documentation.

**Comment by Poeschl** [comment]
Will do this evening.

**Comment by Poeschl** [comment]
Ah, forget the comment before ;) 
* grabs full cup of coffee

**Comment by Poeschl** [comment]
The prettier should now be happy as well

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ The Action provides the following information as output:
 |     `slug`      |                The configured add-on slug.                |
 |    `target`     |                 The add-on target folder.                 |
 |     `image`     |             The Image-template of the add-on.             |
+|    `version`    |                 The version of the addon.                 |
```
Comment (frenck) [inline README.md:79]:
```suggestion
|    `version`    |                 The version of the add-on.                 |
```

File: README.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ The Action provides the following information as output:
 |     `slug`      |                The configured add-on slug.                |
 |    `target`     |                 The add-on target folder.                 |
 |     `image`     |             The Image-template of the add-on.             |
+|    `version`    |                 The version of the add-on.                 |
```
Comment (frenck) [inline README.md:79]:
```suggestion
|    `version`    |                The version of the add-on.                 |
```

**Review by frenck — APPROVED** [review]
Thanks, @Poeschl 👍

**Pull request merged**